sbndcode/BlipRecoSBND/Utils/BlipUtils.cc

```diff
@@ -55,7 +55,9 @@ namespace BlipUtils {
     pinfo.pathLength  = PathLength( part, pinfo.startPoint, pinfo.endPoint);
 
     // Central position of trajectory
-    pinfo.position    = 0.5*(pinfo.startPoint+pinfo.endPoint);
+    pinfo.position.SetXYZ(0.5*(pinfo.startPoint.X()+pinfo.endPoint.X()),
+                         0.5*(pinfo.startPoint.Y()+pinfo.endPoint.Y()),
+                         0.5*(pinfo.startPoint.Z()+pinfo.endPoint.Z()) );
 
```

Comment:
This should be equivalent to:

```suggestion
    pinfo.position = (pinfo.startPoint + pinfo.endPoint)/2;

```